trimChamber.py

```diff
@@ -4,54 +4,64 @@
 By: Christine McLean (ch.mclean@cern.ch), Cameron Bravo (c.bravo@cern.ch), Elizabeth Starling (elizabeth.starling@cern.ch)
 """
 
-import sys, os, random, time
+import sys
 from array import array
-from GEMDAQTestSuite import *
+from macros.fitScanData import fitScanData
 from gempython.tools.vfat_user_functions_uhal import *
-from fitScanData import fitScanData
 
 from qcoptions import parser
 
 parser.add_option("--path", type="string", dest="path", default="data",
                   help="Specify Output File Path", metavar="path")
+parser.add_option("-p", "--ptrim", type="float", dest="ptrim", default=4.0,
+                  help="Specify GEB (long/short)", metavar="GEBtype")
 
+
+uhal.setLogLevelTo( uhal.LogLevel.WARNING )
 (options, args) = parser.parse_args()
 
+ptrim = options.ptrim
+print 'trimming at p = %f'%ptrim
+
+chamber_config = {
+  0:"SC1L1",
+  1:"SC1L2",
+  2:"SC27L1",
+  3:"SC27L2",
+  4:"SC28L1",
+  5:"SC28L2",
+  6:"SC29L1",
+  7:"SC29L2",
+  8:"SC30L1",
+  9:"SC30L2"
+  }
+
+if os.getenv('DATA_PATH') == None or os.getenv('DATA_PATH') == '':
+    print 'You must source the environment properly!'
+if os.getenv('BUILD_HOME') == None or os.getenv('BUILD_HOME') == '':
+    print 'You must source the environment properly!'
+
+
 from ROOT import TFile,TTree,TH1D,TCanvas,gROOT,gStyle,TF1
 import ROOT
```

Comment:
Is this really needed?
If not, i.e., everything you need is handled in the previous line, don't just ```import ROOT```, otherwise, suggest 
```import ROOT as r```
to keep scoping, i.e., ```r.kRed``` rather than ```kRed``` for things that may have name clashes